tests/std/tests/P0896R4_common_iterator/test.cpp

```diff
@@ -194,8 +194,43 @@ bool test_operator_arrow() {
     return true;
 }
 
+struct poor_sentinel {
+    template <weakly_incrementable _Winc>
+    [[nodiscard]] constexpr bool operator==(const _Winc&) const noexcept {
+        return true;
+    }
+
+    template <weakly_incrementable _Winc>
+    [[nodiscard]] constexpr iter_difference_t<_Winc> operator-(const _Winc&) const noexcept {
+        return 0;
+    }
+
+    template <weakly_incrementable _Winc>
+    [[nodiscard]] friend constexpr iter_difference_t<_Winc> operator-(const _Winc&, const poor_sentinel&) noexcept {
+        return 0;
+    }
+};
+
+void test_2065() { // Guard against regression of GH-2065, for which we previously stumbled over CWG-1699.

```

Comment:
This should be either `test_gh_2065` or `test_GH_2065` because we should never mention bug numbers without a database.